src/core/adapters/plane.test.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @jest-environment node
+ */
+
+import { JSDOM } from "jsdom";
+
+import scan from "./plane";
+
+const pages = {
+  issuePage: {
+    url: new URL(
+      "https://app.plane.so/my_team/projects/e9460bd9-2967-4c5e-8d95-24b053e6965d/issues/9210a612-338b-4620-a834-07a89a26bf65/",
+    ),
+    document: "<title>FOO-1 Do something interesting</title><meta property='og:title' content='Plane | Simple, extensible, open-source project management tool.'>",
+  },
+};
+
+describe("plane adapter", () => {
```

Comment:
How about an additional test for a page that's part of the app **but not** the issue page? If there are any, I don't know the app.

Reply:
Yes there are also other pages, but I think checking the title should work well.